examples/pytorch/preprocessing_dist_training/substitute_to_hetero.py

```diff
@@ -0,0 +1,36 @@
+import argparse
+import os
+from collections import defaultdict
+import glob
+
+path = os.getcwd()
+parser = argparse.ArgumentParser()
+parser.add_argument("-n", "--name", help="name of graph to create", default="order")
+parser.add_argument("-p", "--path", help="path where data is stored", default=path)
+parser.add_argument("-f", "--file", nargs="+", default=None)
+parser.add_argument("-r", "--removed", help="file where we have edges that were dropped", default="remove.csv")
+args = parser.parse_args()
+
+graph_name = args.name
+
+if args.file is None:
+    edges_file = sorted(glob.glob(os.path.join(args.path, "*-{}_edges.txt".format(graph_name))))
+
+
+
+for edge_file in edges_file:
+    edges_dict = defaultdict(dict)
+    with open(edge_file) as fp:
+        for line in fp:
+            if line:
+                formatted_line = line.strip()
+                resp_array = formatted_line.split(" ")
+                edges_dict[resp_array[2]][resp_array[3]] = "{} {}".format(resp_array[0], resp_array[1])
+
+    with open(args.removed) as fp, open(edge_file,'a') as wr:
```

Comment:
Code will be optimised If I would have created dict for removed edges and used the same dict for all the parts. I didn't do that because there may be cases where dropped edges are very large and can't fit in RAM,
To handle this made but bit unoptimised.